internal/lefthook/runner/runner.go

```diff
@@ -478,19 +478,21 @@ func (r *Runner) logExecute(name string, err error, out io.Reader) {
 	var execLog string
 	switch {
 	case r.SkipSettings.SkipExecutionInfo():
-		execLog = "\n"
+		execLog = ""
 	case err != nil:
 		execLog = fmt.Sprint(log.Red("\n  EXECUTE > "), log.Bold(name))
 	default:
 		execLog = fmt.Sprint(log.Cyan("\n  EXECUTE > "), log.Bold(name))
 	}
 
-	if err == nil && r.SkipSettings.SkipExecutionOutput() {
+	if execLog != "" {
 		log.Info(execLog)
+	}
+
+	if err == nil && r.SkipSettings.SkipExecutionOutput() {
```

Comment:
These two logs should be switched since we must skip logs execution here.

Reply:
Done :+1: 

So I guess this means that the previous [`log.Info(execLog)`](https://github.com/evilmartians/lefthook/blob/9f47c64faff4c4f417c3dcf578e8bf592c36ae9b/internal/lefthook/runner/runner.go#L489) was not intended to be there - Because it seems that now the behavior just changed so that `execLog` is never printed when the condition `err == nil && r.SkipSettings.SkipExecutionOutput()` is met.

Reply:
Oh god, I'm sorry. `execLog` shouldn't be printed if it is empty. That's right. And if we have SkipExecutionOutput() then we don't print command output. So, your previous solution was correct. Let me fix that.